### DISCUSSION TITLE
Problem in default discussion template

### DISCUSSION THREAD

**Discussion by ahammadshawki8** [community/community#2987] [Discussions]
I saw that there is an extra newline in the default GitHub discussions temple.

  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Was this created intentionally?
Or, should it be like this,
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by becca** [comment]
Thank you for reporting this, it's not intentional and we'll look to resolve this as soon as possible.

**Reply by rajbos** [reply]
Two years later, this is still in there!
